Makefile

```diff
@@ -265,7 +265,6 @@ artifacts: kustomize yq helm ## Generate release artifacts.
 	$(KUSTOMIZE) build config/dev -o artifacts/manifests-dev.yaml
 	$(KUSTOMIZE) build config/alpha-enabled -o artifacts/manifests-alpha-enabled.yaml
 	$(KUSTOMIZE) build config/prometheus -o artifacts/prometheus.yaml
-	$(KUSTOMIZE) build config/visibility -o artifacts/visibility-api.yaml
```

Comment:
For the ease of reviewing I would suggest to merge-in the yaml in a dedicated PR. WDYT @tenzen-y ?

Reply:
Yeah, we can move manifests in a dedicated PR.

Reply:
Created separate PR for this changes https://github.com/kubernetes-sigs/kueue/pull/3084.